publisher/README.md

```diff
@@ -1,92 +1,65 @@
+| nats-pub-creds       | NATS_PUB_CREDS_FILE     | (Y/N*) NATS Publisher Credentials File path (combined JWT and NKey file)  | -                                |
+| nats-pub-jwt         | NATS_PUB_JWT            | (Y/N*) NATS Publisher Credentials JWT string                              | -                                |
+| nats-pub-nkey        | NATS_PUB_NKEY           | (Y/N*) NATS Publisher Credentials NKey string                             | -                                |
+| eth-node-address     | ETH_NODE                | (N) Ethereum Full Node address                                            | -                                |
+| db-host              | DB_HOST                 | (Y) Database host string                                                  | -                                |
+| db-port              | DB_PORT                 | (Y) Database port                                                         | -                                |
+| db-user              | DB_USER                 | (Y) Database User Name                                                    | -                                |
+| db-passw             | DB_PASSWORD             | (Y) Database Password                                                     | -                                |
+| db-name              | DB_NAME                 | (Y) Database Name                                                         | -                                |
+| cache-logs-expire    | LOG_CACHE_EXPIRY_TIME   | (N**) Log Cache Record Expiration Time                                    | 2m                               |
+| cache-logs-purge     | LOG_CACHE_PURGE_TIME    | (N**) Log Cache Record Purge Time                                         | 3m                               |
+| cache-prices-expire  | PRICE_CACHE_EXPIRY_TIME | (N**) Token Price Cache Record Expiration Time                            | 2m                               |
+| cache-prices-purge   | PRICE_CACHE_PURGE_TIME  | (N**) Token Price Cache Record Purge Time                                 | 3m                               |
+| coingecko-api        | COINGECKO_API_URL       | (N**) CoinGecko API url                                                   | https://api.coingecko.com/api/v3 |
+| api-timeout          | API_FETCH_TIMEOUT       | (N**) API fetch timeout                                                   | 2m                               |
+| api-ratelimit        | API_RATE_LIMIT          | (N**) Conservative API Rate Limit(e.g. 10-30 calls per minute)            | 12                               |
+
+*If `nats-sub-creds` (nats creds file location) is set, then `nats-sub-jwt` and `nats-sub-nkey` are not required. Otherwise `nats-sub-jwt` and `nats-sub-nkey` can be set and `nats-sub-creds` has to be empty. The same applies to `nats-pub-*`.</br>
```

Comment:
I think this and the next line miss formatting (a space?)

Reply:
Fixed